docs/overview.md

```diff
@@ -42,7 +42,7 @@ batteries, which are enumerated below.
   provide an API for getting information about transactions we wish to be
   informed of, which is compatible with Electrum server requests/neutrino
   filtering/etc.
-  * [**WIP sample module in Rust**](https://github.com/rust-bitcoin/rust-lightning/pull/791)
+  * [**Sample module in Rust**](https://github.com/rust-bitcoin/rust-lightning/tree/main/lightning-block-sync)
```

Comment:
I wonder if we should `TODO` somewhere to have some `README`'s in each of these sample modules. Otherwise the sample module links like these are a bit oblique.